src/main/java/uk/gov/ons/census/fwmtadapter/model/dto/fwmt/FwmtCreateActionInstruction.java

```diff
@@ -27,5 +27,5 @@
   private boolean handDeliver;
   private Integer ceExpectedCapacity;
   private Integer ceActualResponses;
-  private boolean undeliveredAsAddress; // TODO: This will probably be removed
+  private Boolean undeliveredAsAddress; // TODO: This will probably be removed
```

Comment:
Why this change? We default UAA to false on sample load anyway.

Reply:
it's `Boolean` in the `CollectionCase` source, so it's `Boolean` here in case of garbage in, garbage out. Have removed the comment (which was there before)